index.html

```diff
@@ -417,18 +416,26 @@ <h3>CONTACTS <div class="underline"><span></span></div>
     function myFunction() {
       preloader.style.display = 'none';
     }
+    let navbar = document.getElementById("nav");
+    let navmenu = document.getElementById("navmenu");

```

Comment:
Instead of using `getElementById`, It's better to use `querySelector`, it'd be nice.
Your code will look like a pro...

``` diff
- let navbar = document.getElementById("nav");
- let navmenu = document.getElementById("navmenu");
+ let navbar = document.querySelector("#nav");
+ let navmenu = document.querySelector("#navmenu");
```

Reply:
@Susmita-Dey 
Yup, most of them use querySelector.
If you watch tutorials, they'll use it all the times.

Reply:
yep querySelector is faster than getElementById